src/applications/personalization/profile/tests/components/alerts/Profile.data-unavailable-error.unit.spec.js

```diff
@@ -193,7 +193,7 @@ describe('Profile "Not all data available" error', () => {
     await errorAppearsOnAllPages();
   });
 
-  it('should be shown on all pages if there is an error with the DD4CNP `GET payment_information` endpoint`', async () => {
+  it.skip('should be shown on all pages if there is an error with the DD4CNP `GET payment_information` endpoint`', async () => {
```

Comment:
DD APIs are always going to fail now so we are temporarily hiding the general "we couldn't load all your data" error